homework/CSharpException/Triangle.cs

```diff
@@ -0,0 +1,46 @@
+﻿
+namespace CSharpException
+{
+    public class Triangle
+    {
+        public int OneLength { get; set; }
+        public int TwoLenght { get; set; }
+        public int ThreeLenght { get; set; }
+
+        public Triangle()
+        {
+        }
+        public Triangle(int oneLength, int twoLenght, int threeLenght)
+        {
+            OneLength = oneLength;
+            TwoLenght = twoLenght;
+            ThreeLenght = threeLenght;
+        }
+
+        public void SideLengthCheck(int OneLength, int TwoLenght, int ThreeLenght)
+        {
+            if (OneLength < TwoLenght + ThreeLenght
+                || TwoLenght < OneLength + ThreeLenght
+                || ThreeLenght < OneLength + TwoLenght)
+            {
+                throw new Exception("The length of a side is greater than the sum of the other two!");
+            }
+            else
+            {
+                Console.WriteLine("The length of a side is less than the sum of the other two!");
+            }
+        }
+
+        public void CheckingValuesForZero(int OneLength, int TwoLenght, int ThreeLenght)
+        {
+            while (OneLength != 0
```

Comment:
Wrong implementation. First of all you should check that if at least one number is equal to zero, then throw an exception. This can be done in the if...else construction. Then using do while loop enter the numbers while numbers will be greater than zero.